docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss

```diff
@@ -69,24 +69,17 @@
 
 // Link style for main content that is not a heading
 // The default $icon-width is set for the width of external link icons
-@mixin hb-link--inline($icon: false, $icon-width: 16px, $icon-position: right) {
+@mixin hb-link--inline {
   // no font size by default
   color: var(--palette--tertiary);
 
   @include hb-themes(('colorful', 'airy')) {
-    @include hb-link-background-image($icon, $icon-width, $icon-position);
```

Comment:
I'm pretty sure you're removing all uses of this mixin, you can remove altogether.